packages/firestore/scripts/run-tests.js

```diff
@@ -1,4 +1,4 @@
-"use strict";
+'use strict';
```

Comment:
I know this is a pain, but I am trying not to run these generated sources through formatting to make it obvious that they are generated.